src/System.Drawing.Common/src/mono/System.Drawing.Drawing2D/GraphicsPath.cs

```diff
@@ -0,0 +1,896 @@
+                {
+			if (points == null)
+				throw new ArgumentNullException ("points");
+
+                        Status status = SafeNativeMethods.Gdip.GdipAddPathCurve2I (nativePath, points, points.Length, tension);
+                        SafeNativeMethods.Gdip.CheckStatus (status);                      	
+                }
+                
+                public void AddCurve (PointF [] points, float tension)
+                {
+			if (points == null)
+				throw new ArgumentNullException ("points");
+
+                        Status status = SafeNativeMethods.Gdip.GdipAddPathCurve2 (nativePath, points, points.Length, tension);
+                        SafeNativeMethods.Gdip.CheckStatus (status);                      	
+                }
+
+                public void AddCurve (Point [] points, int offset, int numberOfSegments, float tension)
+                {
+			if (points == null)
+				throw new ArgumentNullException ("points");
+
+                        Status status = SafeNativeMethods.Gdip.GdipAddPathCurve3I (nativePath, points, points.Length,
+                                        offset, numberOfSegments, tension);
+                                        
+			SafeNativeMethods.Gdip.CheckStatus (status);                      	                                       
+                }
+                
+                public void AddCurve (PointF [] points, int offset, int numberOfSegments, float tension)
+                {
+			if (points == null)
+				throw new ArgumentNullException ("points");
+
+                        Status status = SafeNativeMethods.Gdip.GdipAddPathCurve3 (nativePath, points, points.Length,
+                                        offset, numberOfSegments, tension);
+                                        
+			SafeNativeMethods.Gdip.CheckStatus (status);                      	                                       
+                }
+                        
+                public void Reset ()
+                {
+                        Status status = SafeNativeMethods.Gdip.GdipResetPath (nativePath);
+                        SafeNativeMethods.Gdip.CheckStatus (status);                      	
+                }
+
+                public void Reverse ()
+                {
+                        Status status = SafeNativeMethods.Gdip.GdipReversePath (nativePath);
+                        SafeNativeMethods.Gdip.CheckStatus (status);                      	
+                }
+
+                public void Transform (Matrix matrix)
+                {
+			if (matrix == null)
+				throw new ArgumentNullException ("matrix");
+
+                        Status status = SafeNativeMethods.Gdip.GdipTransformPath (nativePath, matrix.nativeMatrix);
+                        SafeNativeMethods.Gdip.CheckStatus (status);                      	
+                }
+                
+		[MonoTODO ("The StringFormat parameter is ignored when using libgdiplus.")]
```

Comment:
Also, this doesn't appear to be ignored by the C# code; if anything is ignoring it, it would seem to be libgdiplus itself.